### PR TITLE
fix(xdebug,docker): accomodate linux differences

### PR DESCRIPTION
**Pull request by damienwebdev**
## Purpose of this pull request

Updated the `XDEBUG_CONFIG` command example to show command syntax for Linux systems.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-development-debug.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-development-debug.md [hunk 1]
```diff
@@ -63,6 +63,11 @@ If you use Microsoft Windows, take the following steps before continuing:
    ```bash
    XDEBUG_CONFIG='remote_host=host.docker.internal remote_port=9002'
    ```
+   
+   On Linux, you should instead use:
```
Comment (meker12) [inline src/cloud/docker/docker-development-debug.md:67]:
```suggestion
   On Linux systems, use the following command instead:
```

File: src/cloud/docker/docker-development-debug.md [hunk 2]
```diff
@@ -63,6 +63,11 @@ If you use Microsoft Windows, take the following steps before continuing:
    ```bash
    XDEBUG_CONFIG='remote_host=host.docker.internal remote_port=9002'
    ```
+   
+   On Linux systems, use the following command instead:
```
Comment (meker12) [inline src/cloud/docker/docker-development-debug.md:67]:
```suggestion
   On Linux systems, use the following command instead:
   
```

File: src/cloud/docker/docker-development-debug.md [hunk 3]
```diff
@@ -63,6 +63,12 @@ If you use Microsoft Windows, take the following steps before continuing:
    ```bash
    XDEBUG_CONFIG='remote_host=host.docker.internal remote_port=9002'
    ```
+
```
Comment (meker12) [inline src/cloud/docker/docker-development-debug.md:66]:
```suggestion

```

File: src/cloud/docker/docker-development-debug.md [hunk 4]
```diff
@@ -63,6 +63,12 @@ If you use Microsoft Windows, take the following steps before continuing:
    ```bash
    XDEBUG_CONFIG='remote_host=host.docker.internal remote_port=9002'
    ```
+   
+   On Linux systems, use the following command instead:
+
```
Comment (meker12) [inline src/cloud/docker/docker-development-debug.md:68]:
```suggestion

```

**Comment by meker12** [comment]
running tests

**Pull request merged**